types/crew.model.ts

```diff
@@ -0,0 +1,64 @@
+import { Club } from './club.model';
+import { DisplayStatus } from './display-status.model';
+import { Event } from './event.model';
+import { Result } from './result.model';
+import { Rower } from './rower.model';
+
+export interface Crew {
+	id: string;
+	club_id: string;
+	regatta_id: string;
+	event_id: string;
+	sub_event_id: null; // Wat is een sub_event?
```

Comment:
Dit is een string|null:

Een ploeg kan een sub_event_id hebben dit treed op als er te weinig ploegen zijn voor een eigen veld. Dus ze starten dan door elkaar maar willen wel een eigen uitslag hebben. (Er zitten wat meer regels aan)